src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -210,14 +192,25 @@ public void Prepare(CancellationToken token)
                 Debug.Assert(sourceControlledFileSystem.FileExistOrQueuedToBeWritten(info.FilePath), "Expected a rule set to pend pended");
             }
 
-            foreach (var binder in this.childBinder)
+            foreach (var project in projects)
             {
                 if (token.IsCancellationRequested)
                 {
                     return;
                 }
-
-                binder.Prepare(token);
+                if (projectBinderFactory.Get(project) is RoslynProjectBinder)
```

Comment:
This is the reason for the change:
I will change this code to be 
```
var projectBinder = projectBinderFactory.Get(project);
projectBinder.Bind(project, bindingConfigFile, token);
```
And the roslyn implementation will create ProjectBindingOperation and call it.